examples/active-class-name/package.json

```diff
@@ -8,7 +8,8 @@
   "dependencies": {
     "next": "latest",
     "react": "18.1.0",
-    "react-dom": "18.1.0"
+    "react-dom": "18.1.0",
+    "styled-jsx": "5.0.2"
```

Comment:
styled-jsx shouldn't need to be manually installed, is there a related issue for this?

Reply:
Might be `pnpm` related, this failed for me when trying to boot up locally. :thinking:

I needed to do the same in #37712. See: https://github.com/vercel/next.js/pull/37712/files#diff-53c991f1117ac296230d2d48db9febf631f444ddac85ae3c26310019194539fbR31

Reply:
Yeah it seems the types fail to resolve for `styled-jsx` because it's a nested dependency of Next.js and isn't hoisted with `pnpm`. We need to fix this in Next.js so that users don't need to install `styled-jsx` manually 🤔